File: AppGenerateFiles/dbo/Model/Transaction_Contratos_old.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Transaction_Contratos_old : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? numero_contrato { get; set; }
       public DateTime? fecha_contrato { get; set; }
       public DateTime? fecha_cancelar { get; set; }
       public Double? monto { get; set; }
       public Double? interes { get; set; }
       public Double? mora { get; set; }
       public string? estado { get; set; }
       public DateTime? fecha_vencimiento { get; set; }
       public int? codigo_cliente { get; set; }
       public Double? saldo { get; set; }
       public int? dias_mora { get; set; }
       public Double? saldo_mora { get; set; }
       public DateTime? fecha_baja { get; set; }
       public Double? abonos { get; set; }
       public DateTime? ultima_visita { get; set; }
       public int? tipo { get; set; }
       public string? entregado { get; set; }
       public Double? interes_actual { get; set; }
       public string? observaciones { get; set; }
       public Double? iva { get; set; }
       public Double? margen { get; set; }
       public Double? interesl { get; set; }
       public Double? moral { get; set; }
       public Double? descuento { get; set; }
       public Double? util { get; set; }
       public Double? taza_interes_cargos { get; set; }
       public Double? taza_mora { get; set; }
       public DateTime? fecha_mora { get; set; }
       public DateTime? fecha_interes { get; set; }
       public Double? taza_gestion_crediticia { get; set; }
       public int? Id_User_OLD { get; set; }
       public Double? taza_cambio { get; set; }
       public int? id_agente { get; set; }
       public Double? dkm { get; set; }
       public Double? gasolinamonto { get; set; }
       public Double? valorcad { get; set; }
       public int? plazo { get; set; }
       public Double? cuotafija { get; set; }
       public Double? montocuotaatrazadas { get; set; }
       public DateTime? mes_pagado { get; set; }
       public Double? tasa_hoy { get; set; }
       public int? numero_protocolo { get; set; }
       public Double? valor_dolar { get; set; }
       public Double? parciales { get; set; }
       public Double? mora_parcial { get; set; }
       public Double? interes_parcial { get; set; }
       public string? motivo_anulacion { get; set; }
       public int? idcatemp { get; set; }
       public Double? cuota_fija_inicial { get; set; }
       public DateTime? fecha_cancelar_inicial { get; set; }
       public int? plazo_inicial { get; set; }
       public int? dias_para_baja { get; set; }
       public int? Id_User { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Transaction_Contratos_old.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools, BasicStates } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
class Transaction_Contratos_old extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ numero_contrato;
   /**@type {Date}*/ fecha_contrato;
   /**@type {Date}*/ fecha_cancelar;
   /**@type {Number}*/ monto;
   /**@type {Number}*/ interes;
   /**@type {Number}*/ mora;
   /**@type {String}*/ estado;
   /**@type {Date}*/ fecha_vencimiento;
   /**@type {Number}*/ codigo_cliente;
   /**@type {Number}*/ saldo;
   /**@type {Number}*/ dias_mora;
   /**@type {Number}*/ saldo_mora;
   /**@type {Date}*/ fecha_baja;
   /**@type {Number}*/ abonos;
   /**@type {Date}*/ ultima_visita;
   /**@type {Number}*/ tipo;
   /**@type {String}*/ entregado;
   /**@type {Number}*/ interes_actual;
   /**@type {String}*/ observaciones;
   /**@type {Number}*/ iva;
   /**@type {Number}*/ margen;
   /**@type {Number}*/ interesl;
   /**@type {Number}*/ moral;
   /**@type {Number}*/ descuento;
   /**@type {Number}*/ util;
   /**@type {Number}*/ taza_interes_cargos;
   /**@type {Number}*/ taza_mora;
   /**@type {Date}*/ fecha_mora;
   /**@type {Date}*/ fecha_interes;
   /**@type {Number}*/ taza_gestion_crediticia;
   /**@type {Number}*/ Id_User_OLD;
   /**@type {Number}*/ taza_cambio;
   /**@type {Number}*/ id_agente;
   /**@type {Number}*/ dkm;
   /**@type {Number}*/ gasolinamonto;
   /**@type {Number}*/ valorcad;
   /**@type {Number}*/ plazo;
   /**@type {Number}*/ cuotafija;
   /**@type {Number}*/ montocuotaatrazadas;
   /**@type {Date}*/ mes_pagado;
   /**@type {Number}*/ tasa_hoy;
   /**@type {Number}*/ numero_protocolo;
   /**@type {Number}*/ valor_dolar;
   /**@type {Number}*/ parciales;
   /**@type {Number}*/ mora_parcial;
   /**@type {Number}*/ interes_parcial;
   /**@type {String}*/ motivo_anulacion;
   /**@type {Number}*/ idcatemp;
   /**@type {Number}*/ cuota_fija_inicial;
   /**@type {Date}*/ fecha_cancelar_inicial;
   /**@type {Number}*/ plazo_inicial;
   /**@type {Number}*/ dias_para_baja;
   /**@type {Number}*/ Id_User;
}
export { Transaction_Contratos_old }
